src/components/players/validators.ts

```
@@ -322,3 +360,53 @@ export const validatePlayerUpdateRequest = () =>
+    tracking_number: {
+      in: ["body"],
+      optional: true,
+      isString: true,
+      // errorMessage: "tracking_number is required",
+    },
+    amount: {
+      in: ["body"],
+      optional: true,
+      isNumeric: true,
+      custom: {
+        options: (val) => !isNaN(Number(val)),
```

Biome (1.9.4)

[error] isNaN is unsafe. It attempts a type coercion. Use Number.isNaN instead.

See the MDN documentation for more details.
Unsafe fix: Use Number.isNaN instead.


(lint/suspicious/noGlobalIsNan)